tests/utility/v2/test_component_analyses.py

```diff
@@ -15,19 +15,28 @@
 # Author: Deepak Sharma <deepshar@redhat.com>
 #
 """Test Component Analyses Utility Stand."""
-
+import json
+import os
 import unittest
 from unittest.mock import patch
 
 from werkzeug.exceptions import BadRequest
 
 from bayesian.utility.v2.component_analyses import validate_version, \
-    known_package_flow, ca_validate_input
+    known_package_flow, ca_validate_input, get_known_unknown_pkgs
 
 
 class TestComponentAnalyses(unittest.TestCase):
     """Test Component Analyses Utility class."""
 
+    @classmethod
+    def setUpClass(cls) -> None:
+        """Init Test class."""
+        batch_data_no_cve = os.path.join('/bayesian/tests/data/gremlin/batch_data_no_cve.json')
+
+        with open(batch_data_no_cve) as f:
+            cls.gremlin_batch_data_no_cve = json.load(f)
```

Comment:
Why this has to be hanging around at the class level where batch_data_no_cve is been used only in test_get_known_unknown_pkgs_no_cve?

Reply:
Ack!